plugins/org.jkiss.dbeaver.ext.oracle/OSGI-INF/l10n/bundle.properties

```diff
@@ -181,6 +181,12 @@ meta.org.jkiss.dbeaver.ext.oracle.model.OracleObject.name.name=Name
 meta.org.jkiss.dbeaver.ext.oracle.model.OraclePackage.valid.name=Valid
 meta.org.jkiss.dbeaver.ext.oracle.model.OraclePackage.objectDefinitionText.name=Header
 meta.org.jkiss.dbeaver.ext.oracle.model.OraclePackage.extendedDefinitionText.name=Body
+meta.org.jkiss.dbeaver.ext.oracle.model.OraclePackage.created.name=Created
+meta.org.jkiss.dbeaver.ext.oracle.model.OraclePackage.created.description=Timestamp for the creation of the object
+meta.org.jkiss.dbeaver.ext.oracle.model.OraclePackage.temporary.name=Temporary
+meta.org.jkiss.dbeaver.ext.oracle.model.OraclePackage.temporary.description=Whether the object is temporary (the current session can see only data that it placed in this object itself)
+meta.org.jkiss.dbeaver.ext.oracle.model.OraclePackage.lastDDLTime.name=Last DDL Time
+meta.org.jkiss.dbeaver.ext.oracle.model.OraclePackage.lastDDLTime.description=Timestamp for the last modification of the object resulting from a DDL statement (including grants and revokes)
```

Comment:
I don't like `resulting`. For me, it makes sentence unclear. Maybe `due to` or `as a result` fits better.